07/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device=width, initial-scale=1.0">
+    <meta name="description" content="Venta de motos suzuki, Refacciones, mantenimientos y servicios correcctivos">
+    <meta name="author" content="Jesus David Ceballos Parra">
+    <title>Suzuki Motos Tocino</title>
+  </head>
+  <body>
+    <header>
+      <h1>Suzuki Motos</h1>
+      <ul>
+        <li><a href="#intro">Acerca de Suzuki motosa</a></li>
+        <li><a href="#Gmodelos">Modelos</a></li>
+        <li><a href="#servicio">Servicios</a></li>
+        <li><a href="#contacto">Comunicate con nostros</a></li>
```

Comment:
You should improve the distribution of the tags
---
Deberías mejorar la distribución de las etiquetas

```
<li>
  <a>
    Link
  </a>
</li>
```